verifier/model.py

```diff
@@ -5,8 +5,8 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import pytorch_lightning as pl
-import torchmetrics
 from transformers import AutoTokenizer, AutoModel
+from torchmetrics.classification import BinaryAccuracy, BinaryAveragePrecision, BinaryF1Score, BinarySpecificity, BinaryRecall, BinaryPrecision
```

Comment:
This line is a bit too long. Could you format the code using [black](https://black.readthedocs.io/en/stable/)? Just install black using pip or conda and then run `black PATH_TO_PY_FILE`.